### PR TITLE
 Fix [UI] - documentation menu hard to close on mobile 

### LINKED ISSUE #1093: [UI] documentation menu is hard to close on mobile
STR:

1. Navigate to https://taskcluster-ui.herokuapp.com/docs/resources
2. Open the developer tools and click on the responsive design mode button

Screen Shot 2019-07-22 at 11 36 54 AM

3. Refresh the page
4. Click the hamburger menu to open the side drawer

Actual: It's hard to close the menu for users on mobile
Expected: It's easy to close the menu for users on mobile

Solution: Make the drawer smaller on mobile when it's open.

### PR DESCRIPTION
**Pull request by rbrishabh**
Fixes #1093 
@helfi92 

Before : Documentation menu was hard to close

After: Documentation button is easy to close

See image:
image

Let me know if these needs any changes.
Thanks! :)

### CODE REVIEW

File: ui/src/components/Dashboard/index.jsx [hunk 1]
```diff
@@ -84,6 +84,9 @@ import SkipNavigation from '../SkipNavigation';
     },
     docsDrawerPaper: {
       width: theme.docsDrawerWidth,
+      [theme.breakpoints.down('xs')]: {
+        width: '240px',
```
Comment (helfi92) [inline ui/src/components/Dashboard/index.jsx]: `width: theme.spacing.unit * 30,`

Comment (rbrishabh) [reply ui/src/components/Dashboard/index.jsx]: Alright, got it! Thanks =)

**Comment by rbrishabh** [comment]
As always, thanks to you, I'm learning a lot!

If I can be of service anywhere else, just tag me! =)

**Comment by helfi92** [comment]
My pleasure! There's a lot of issues/bugs in https://codetribute.mozilla.org/projects/taskcluster. Have a look and see if there's any that interests you.

**Pull request merged**